### ISSUE TITLE
Is there anyway to modify the scrip so it updates the cells by the event title?

### ISSUE THREAD

**Issue by jolesn** [Davepar/gcalendarsync#23]
I noticed that this script is capable of updating a cell if a change to an event was made. I think it is making that update by the event id. Is there anyway to change it so that it updates by event title?

Let's say this is how my columns look like:

           name,            person of contact,    project,      date,                      next action.
          Facebook,       Zuckerberg,              ABC          m-d-y H:m               call and say hi

Now my google calendar is shared with other people and we have to put our initials at the beginning of each event. So let's say my Google Calendar entry looks like this:

Title:JO Facebook
Date: 6/26/2017 15:00:00	6/26/2017 17:30:00

Description: schedule a meeting.

Is there any way to ignore the initials, meaning read the title after the first space. And then do a if "title = name" update date and next action.

I am not sure how difficult of a request this is, I have been trying to modify your code so that it would update by title, it's just beyond my comprehension. Advice on a good tutorial would also be really appreciated.

**Comment by Davepar** [comment]
Correct. The script uses the event ID to sync changes between a calendar and spreadsheet. The ID is what the Google Apps Script Calendar API uses to identify events.  I don't see any API methods that use the title to retrieve or update events, so I don't think that's possible.

Here's the documentation on the API:
https://developers.google.com/apps-script/reference/calendar/

Dave
